bundles/core/src/main/java/com/adobe/cq/wcm/core/components/sandbox/models/Teaser.java

```diff
@@ -28,6 +31,20 @@
  */
 public interface Teaser extends ComponentExporter {
 
+    /**
+     * Name of the property that will store if the teaser has Call-to-Action elements
```

Comment:
I'd write it in present form: 
Name of the property that stores if the teaser has Call-to-Action elements